src/devices/Directory.Build.props

```diff
@@ -13,12 +13,18 @@
     <MainLibraryPath>$(MSBuildThisFileDirectory)../System.Device.Gpio/</MainLibraryPath>
     <SystemDeviceModelProjectName>System.Device.Model</SystemDeviceModelProjectName>
     <SystemDeviceModelPath>$(MSBuildThisFileDirectory)$(SystemDeviceModelProjectName)/$(SystemDeviceModelProjectName).csproj</SystemDeviceModelPath>
+    <CommonProjectName>Common</CommonProjectName>
+    <CommonProjectPath>$(MSBuildThisFileDirectory)$(CommonProjectName)/$(CommonProjectName).csproj</CommonProjectPath>
     <LangVersion>9</LangVersion>
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(MSBuildProjectName)' != '$(CommonProjectName)' AND '$(MSBuildProjectName)' != '$(SystemDeviceModelProjectName)'">
+  <!-- These projects are included into every binding. They must not depend on each other, or we get a circular dependency -->
+    <ProjectReference Include="$(SystemDeviceModelPath)" />
```

Comment:
Usually when we we use infrastructure to automatically Opt-in all of the child projects into, it is good practice to provide an Opt-out switch, for example, a property like `Condition="'$(OmitCommonProjectReference)' != 'true'"` or something like that. If you do that, then instead of having to use the Condition you have above in the ItemGroup, you can just set those properties onto the Common and System.Device.Model projects themselves which would remove the ProjectReference from there. You don't have to make that change now, and I'm fine to keeping this as is, just wanted to share that other point of view.